docs/Contributors_Guide/add_use_case.rst

```diff
@@ -211,14 +211,14 @@ use case OR category directory for a model_applications use case
 
     * Update the list of External Dependencies (if applicable) to include any
       required Python packages.  Update the
-      `METplus  Components Python Requirements <https://docs.google.com/spreadsheets/d/1Lf-yxZBXTTnBaqCOWUk-jdP9RpaLfil_s-KKH29CkKU/edit#gid=0>`_
-      spreadsheet.  If the package is already listed in the spreadsheet, add
+      `METplus Components Python Requirements <https://metplus.readthedocs.io/en/develop/Users_Guide/overview.html#metplus-components-python-requirements>`_
```

Comment:
Shouldn't this be a relative path instead of always using develop?

Reply:
@georgemccabe Great suggestion (via slack) to make it a label.  Thanks!  It is now a label.  Please take a look at the Files Changed and the [documentation](https://metplus.readthedocs.io/en/feature_1403_python_components_table/Contributors_Guide/add_use_case.html#add-sphinx-documentation-file) itself.